addons/web/static/src/js/widgets/debug_manager.js

```diff
@@ -148,14 +148,12 @@ if (core.debug) {
             });
         },
         translate: function() {
-            this.do_action({
-                name: _t("Technical Translation"),
-                res_model : 'ir.translation',
-                domain : [['type', '!=', 'object'], '|', ['name', '=', this.dataset.model], ['name', 'ilike', this.dataset.model + ',']],
-                views: [[false, 'list'], [false, 'form']],
-                type : 'ir.actions.act_window',
-                view_type : "list",
-                view_mode : "list"
+            var self = this;
+            session.rpc('/web/webclient/technical_translations', {
+                model: this.dataset.model
+            }).then(function(result) {
+                result.views = [[false, "list"], [false, "form"]];
+                return self.do_action(result);
             });
```

Comment:
I think that the parameter `views` is not useful here, and its value looks wrong to me.
Moreover, this parameter should be returned by the controller. This would simplify the code to be

    }).then(this.do_action);

(Note that methods starting with `"do_"` are bound correctly to `this`.)